src/plugins/intel_gpu/tests/unit/test_cases/col2im_gpu_test.cpp

```diff
@@ -17,7 +17,7 @@ using namespace ::tests;
 static void test_col2im_output(bool is_caching_test) {
     auto& engine = get_test_engine();
 
```

Comment:
do we need to keep this unittest? If func_test is sufficient, I think we can remove this test.

Reply:
Removed.